### PR TITLE
Document default room creator PL

### PR DESCRIPTION
**Pull request by maxidor**
Follow up of #1052

### CODE REVIEW

File: event-schemas/schema/m.room.power_levels [hunk 1]
```diff
@@ -8,7 +8,8 @@ description: |-
   If a ``user_id`` is in the ``users`` list, then that ``user_id`` has the
   associated power level. Otherwise they have the default level
   ``users_default``. If ``users_default`` is not supplied, it is assumed to be
-  0.
+  0. If the room contains no ``m.room.power_levels`` event, its creator gets a
```
Comment (richvdh) [inline event-schemas/schema/m.room.power_levels]: s/its/the room's/ ?

Comment (richvdh) [reply event-schemas/schema/m.room.power_levels]: `PL` isn't used elsewhere in the spec: it is an informal term and I think it's best to stick with the full "power level" in the spec.

Comment (richvdh) [reply event-schemas/schema/m.room.power_levels]: sorry I'm about to completely rewrite this sentence, but I think we probably want:

> If the room contains no ``m.room.power_levels`` event, the room's creator has a power level of 100, and all other users have a power level of 0.

**Comment by maxidor** [comment]
@richvdh changed wording as per your feedback!

File: event-schemas/schema/m.room.power_levels [hunk 2]
```diff
@@ -8,7 +8,8 @@ description: |-
   If a ``user_id`` is in the ``users`` list, then that ``user_id`` has the
   associated power level. Otherwise they have the default level
   ``users_default``. If ``users_default`` is not supplied, it is assumed to be
-  0.
+  0. If the room contains no m.room.power_levels event, the room's creator has
```
Comment (richvdh) [inline event-schemas/schema/m.room.power_levels]: backticks please

sorry :/

**Comment by maxidor** [comment]
done, you might want to squash this PR when you merge it :)

**Comment by richvdh** [comment]
thanks!

**Pull request merged**